docusaurus/src/pages/index.js

```diff
@@ -37,40 +43,113 @@ export default function Home() {
 
     return (
         <OpenZitiLayout>
-            <div style={{display:"flex", alignItems:"center", flexDirection:"column"}}>
-                <ThemedImage
-                    alt="OpenZiti Logo"
-                    style={{paddingTop: "75px", paddingBottom: "10px", width:"50%", alignSelf: "center", minWidth: "400px"}}
-                    sources={{
-                        light: useBaseUrl('/img/ziti-logo-dark.svg'),
-                        dark: useBaseUrl('/img/ziti-logo-light.svg'),
-                    }}
-                />
-                <div style={{display:"flex", justifyContent:"center"}}>
-                <p style={{fontSize: "x-large"}}>-- Replacing Infrastructure With Software --</p>
+            <OpenZitiHorizontalSection style={{padding: "0px"}}>
+                <div className={styles.titleSection} style={{backgroundColor: "var(--callout-color)"}}>
+                    <H2>Application Embedded Zero Trust</H2>
+
+                    <p>OpenZiti is taking application and API security to the  to making secure connectivity simple and accessible by <Highlight>replacing
```

Comment:
This paragraph sounds very "markety" to me (especially words like "ground-breaking").  Might be better off with something more terse (I think at one point you had something like "solve the challenges of secure connectivity with software").  Further on in the docs we can delve into why this is a great idea / why somebody should want to do it

Reply:
or maybe just leave "The world is software, your network security needs to react with the speed of software; it needs to be software" if we want to keep some of that vibe...